integration-tests/src/test/groovy/com/okta/sdk/tests/it/UserTypesIT.groovy

```diff
@@ -15,7 +15,7 @@
  */
 package com.okta.sdk.tests.it
 
-import com.okta.sdk.resource.UserType
+import com.okta.sdk.resource.user.type.UserType
```

Comment:
user.type seems a bit redundant, maybe just tag it with `user`?
(maybe there are a bunch of other user type objects 🤷)

Reply:
Agreed :) 

`UserType` and `UserTypeList` are the only classes in package `com.okta.sdk.resource.user.type`.

<img width="346" alt="image" src="https://user-images.githubusercontent.com/61501885/84931823-32a5f800-b088-11ea-8e2d-2dabee46e0c4.png">

Will consider moving it to a different package (say `com.okta.sdk.resource.user`) as the `user.type` seems redundant.  Will do this as part of separate PR.

Reply:
https://oktainc.atlassian.net/browse/OKTA-305898 created to address this improvement.